hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/SecureBulkLoadManager.java

```diff
@@ -291,10 +291,10 @@ private boolean isUserReferenced(UserGroupInformation ugi) {
             //We call bulkLoadHFiles as requesting user
             //To enable access prior to staging
             return region.bulkLoadHFiles(familyPaths, true,
-                new SecureBulkLoadListener(fs, bulkToken, conf), request.getCopyFile(),
+              new SecureBulkLoadListener(fs, bulkToken, conf), request.getCopyFile(),
               clusterIds, request.getReplicate());
           } catch (Exception e) {
-            LOG.error("Failed to complete bulk load", e);
+            throw new IOException(e);
```

Comment:
Exception cannot be simply swallowed, this will make client fall into possible infinite retry;